content/medo_manual/werken_met_deelnemers.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Werken met deelnemers
+sort: 300
+---
+
+* TOC
+{:toc}
+
+## Deelnemerslijst
+
+Op deze pagina bevindt zich een tabel met de deelnemers van het project. Door
+op de tabelkoppen te klikken kan de tabel gesorteerd worden. Daarnaast is het
+mogelijk om door de tabel te zoeken, waarbij door data van meerdere kolommen
+gezocht kan worden. Verzamelaars en coördinatoren kunnen de voornaam,
```

Comment:
Als u verzamelaar of coördinator bent kunt u de voornaam, achternaam en het e-mailadres van deelnemers bekijken.